src/ng-select/themes/bootstrap4.theme.scss

```diff
@@ -0,0 +1,381 @@
+@import "bootstrap/scss/bootstrap";
+@import "@ng-select/ng-select/scss/mixins";
+
+.form-control.ng-invalid {
+  @extend .form-control, .is-invalid;
+}
+
+.default-theme {
+  @import "@ng-select/ng-select/scss/default.theme.scss";
+}
+
+//$ng-select-highlight:  #007eff !default;
+$ng-select-primary-text: $input-color !default;
+$ng-select-disabled-text: $text-muted !default;
+$ng-select-border: $input-border-color !default;
+$ng-select-bg: $input-bg !default;
+$ng-select-selected: $dropdown-link-active-bg !default;
+$ng-select-marked: $dropdown-link-active-bg !default;
+
+.bootstrap-theme {
+
+  .ng-select {
+      &.form-control {
+        padding: 0;
+      }
```

Comment:
I suggest to add
```
   &.ng-invalid{
          background-position: right calc(1.375em + .1875rem) center;
        }
```
I know it won't be bootstrap compliant but it will fix one of the issue that is currently there.
https://angular-bs4-ng-select-faf6ul.stackblitz.io

Reply:
Sure changing it

Reply:
Done